security-analysis/security-analysis-api/src/test/resources/ActionFileTest.json

```diff
@@ -30,27 +30,27 @@
     "type" : "PHASE_TAP_CHANGER_TAP_POSITION",
     "id" : "id5",
     "transformerId" : "transformerId1",
-    "value" : 5,
+    "tapPosition" : 5,
```

Comment:
This a breaking change, we accept to fail on old file parsing?

Reply:
@annetill @sylvlecl

Reply:
I can maintain compatibility but not sure it is useful

Reply:
Not sure too, just asking.

Reply:
I think it's a guarantee we must provide from the start and stick with it, unfortunately for the implementation work, and although this breaking change would not be an issue for our own project.

Reply:
ok so for the deserializer I add the possibility to have value ? that's it ?

Reply:
Yes but we must also increment the version format, if it has not been since last release.
(otherwise, we will produce files that will produce unclear errors with older versions of pypowsybl)